src/tests/futuramaData.js

```diff
@@ -0,0 +1,29 @@
+export const futuramaApiData = [
+  {
+    character: 'Fry',
+    quote:
+      'Not everyone turns out like their parents. Look at me: my parents were honest, hard-working people.',
+    image:
+      'https://res.cloudinary.com/dzxqhkyqd/image/upload/v1554904133/fry.png',
+  },
+  {
+    character: 'Fry',
+    quote:
+      "Wow, there's a million aliens! I've never seen something so mind-blowing! Ooh, a reception table with muffins!",
+    image:
+      'https://res.cloudinary.com/dzxqhkyqd/image/upload/v1554904133/fry.png',
+  },
+  {
+    character: 'Leela',
+    quote:
+      "Just make a simple cake. And this time, if someone's going to jump out of it make sure to put them in *after* you cook it.",
+    image:
+      'https://res.cloudinary.com/dzxqhkyqd/image/upload/v1554904145/leela.png',
+  },
+  {
+    character: 'Professor Farnsworth',
+    quote: 'You may feel a slight stinging sensation ...all of you.',
+    image:
+      'https://res.cloudinary.com/dzxqhkyqd/image/upload/v1554904014/farnsworth.png',
+  },
+];
```

Comment:
It is a pretty common naming convention to keep your mock data for testing in a `fixtures` folder.

Reply:
I see. Thanks for clarifying!